packaging/rhel/xrootd.spec.in

```diff
@@ -733,6 +733,10 @@ fi
 %{_libdir}/libXrdN2No2p-4.so
 %{_libdir}/libXrdOssSIgpfsT-4.so
 %{_libdir}/libXrdServer.so.*
+%{_libdir}/libXrdSsi-4.so.*
```

Comment:
Yes, I will give this another go. There is no reason for it not being part of server libs like the other libs.